### PR TITLE
DESENG-481: Adding unit tests for met api

### PR DESCRIPTION
**Pull request by VineetBala-AOT**
Issue #: https://apps.itsm.gov.bc.ca/jira/browse/DESENG-481

*Description of changes:*
- Added new unit tests for met api

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the met-public license (Apache 2.0).

### CODE REVIEW

File: met-api/src/met_api/services/engagement_service.py [hunk 1]
```diff
@@ -56,7 +56,8 @@ def get_engagement(self, engagement_id) -> EngagementSchema:
 
             engagement = EngagementSchema().dump(engagement_model)
             engagement['banner_url'] = self.object_storage.get_url(engagement['banner_filename'])
-        return engagement
+            return engagement
+        return None
```
Comment (NatSquared) [inline met-api/src/met_api/services/engagement_service.py:60]: Why explicitly return `None`? This should not affect functionality; is this for legibility reasons?

File: met-api/src/met_api/utils/enums.py [hunk 2]
```diff
@@ -21,6 +21,7 @@ class ContentType(Enum):
     JSON = 'application/json'
     FORM_URL_ENCODED = 'application/x-www-form-urlencoded'
     PDF = 'application/pdf'
+    FORM = 'application/x-www-form-urlencoded'
```
Comment (NatSquared) [inline met-api/src/met_api/utils/enums.py:24]: Why are we creating `FORM` with the same value as `FORM_URL_ENCODED`? Do we plan to change this later or can we reuse `FORM_URL_ENCODED`

Comment (VineetBala-AOT) [reply met-api/src/met_api/utils/enums.py]: Thanks for catching this. I have updated the code to use FORM_URL_ENCODED.

File: met-api/src/met_api/services/engagement_service.py (hunk 1 above)
Comment (VineetBala-AOT) [reply met-api/src/met_api/services/engagement_service.py:60]: This is just for readability.

**Review by NatSquared — APPROVED** [review]
This PR looks good! Thanks for putting in the work to create all these test scenarios.

**Pull request merged**